medusa/providers/torrent/json/t411.py

```diff
@@ -121,9 +138,12 @@ def parse(self, data, mode):
 
         unsorted_torrent_rows = data.get('torrents') if mode != 'RSS' else data
 
-        if not unsorted_torrent_rows or not isinstance(unsorted_torrent_rows, dict):
-            logger.log('Data returned from provider does not contain any {0}torrents'.format(
-                'confirmed ' if self.confirmed else ''), logger.DEBUG)
+        if not unsorted_torrent_rows:
+            logger.log(
+                'Data returned from provider does not contain any {torrents}'.format(
+                    torrents='confirmed torrents' if self.confirmed else 'torrents'
+                ), logger.DEBUG
+            )
```

Comment:
Change it back if you prefer.  I personally think this is clearer, but it makes no real difference to me.